src/Item_Cluster.php

```diff
@@ -178,6 +190,42 @@ public static function showItems(Cluster $cluster)
         }
     }
 
+    /**
+     * Print the item's operating system form
+     *
+     * @param CommonDBTM $item Item instance
+     *
+     * @since 9.2
+     *
+     * @return void
+     **/
```

Comment:
Doc seems pasted from somewhere else, you can probably delete it anyway as the method is private.

```suggestion
    /**
     * Print the item's operating system form
     *
     * @param CommonDBTM $item Item instance
     *
     * @since 9.2
     *
     * @return void
     **/
```